source/includes/_endpoint-dataset.md

```diff
@@ -686,96 +686,66 @@ Refer to the options described on the table above for the `csv` format to change
 
 
 ##### Match
-`GET /datasets/match{?datasets={id}&datasets={id}[&datasets={id}]+`
+```http
+POST /datasets/match HTTP/1.1
```

Comment:
This needs the trailing slash, else cherrypy will 301 on it if someone was to follow this example